below/model/src/cgroup.rs

```diff
@@ -32,6 +32,9 @@ pub struct SingleCgroupModel {
     #[queriable(preferred_name = mem)]
     pub memory: Option<CgroupMemoryModel>,
     #[queriable(subquery)]
+    #[queriable(preferred_name = pids)]
+    pub pids: Option<CgroupPidsModel>,
```

Comment:
Can we move `pids.max` to the `CgroupProperties` since it is a configured property? I'm happy to keep a single field CgroupPidsModel around

Reply:
Thanks, fixed.